Cargo.toml

```diff
@@ -57,9 +58,10 @@ serde_json = { version = "1.0" }
 tiny-keccak = { version = "1.4" }
 
 [profile.release]
-opt-level = 3
+opt-level = "z"
```

Comment:
Can you help me understand the purpose of changing `opt-level` and `codegen-units` here?

Reply:
Changes reverted.